test-components/ts-rpc/components-ts/caller/golem.yaml

```diff
@@ -6,13 +6,7 @@
 components:
   caller:
     template: ts
-    componentWasm: dist/caller-ts.wasm
-    linkedWasm: ../dist/caller-composed-ts.wasm
-    customCommands:
-      copy:
-      - command: cp dist/caller-ts.wasm ../..
-      - command: cp ../dist/caller-composed-ts.wasm ../..
 dependencies:
   caller:
   - target: counter
-    type: static-wasm-rpc
\ No newline at end of file
+    type: wasm-rpc
```

Comment:
all of these were using static rpc and custom commands deliberately: to be able to test both static and non-static rpc with 1 build. we might have to change this a bit (e.g. because of the new meta data step), but this way it will break the static rpc tests.
cc @vigoo

Reply:
Makes sense. I missed the copying step for the composed version of the components.
There is a different problem with this, will explain during standup